service/src/test/java/uk/gov/hmcts/reform/fpl/utils/JudgeAndLegalAdvisorHelperTest.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.fpl.enums.JudgeOrMagistrateTitle;
+import uk.gov.hmcts.reform.fpl.model.common.JudgeAndLegalAdvisor;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class JudgeAndLegalAdvisorHelperTest {
+    @Test
+    public void legalAdvisorNameShouldBeEmptyWhenJudgeAndLegalAdvisorNull() {
```

Comment:
Still would like to see these start with 'Should' rather than the field.

Reply:
Other tests have the fieldname at the start so I'm going to keep it like this for consistency + readability